README.md

```diff
@@ -1,207 +1,225 @@
 # SignalFx Java Agent
 
-This is the SignalFx Java Agent, a [JVM
-agent](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
-to automatically instrument your Java application to capture and report
-distributed traces to SignalFx. Enabling the agent is done by downloading the
-[latest JAR](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
-and adding its path to your JVM startup options:
+The SignalFx Java Agent is a
+[Java Virtual Machine](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
+(JVM) agent that automatically instruments your Java application to capture and
+report distributed traces to SignalFx. Download the JAR for the agent's 
+[latest release](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
+and add its path to your JVM startup options:
 
 ```bash
-$ # Specify the Java Agent with the appropriate option and launch your application as usual
-$ java -javaagent:signalfx-tracing.jar -jar app.jar
+$ java -javaagent:path/to/signalfx-tracing.jar -jar app.jar
 ```
+For more information, see [Configure the SignalFx Java Agent](#Configure-the-SignalFx-Java-Agent).
 
-In addition to using bytecode manipulation to instrument supported libraries and
-frameworks, the SignalFx Java Agent configures an OpenTracing-compatible tracer
-to capture and export trace spans. It registers this tracer as the OpenTracing
-`GlobalTracer` to easily enable custom instrumentation throughout your
-application:
+The agent instruments supported libraries and frameworks with bytecode
+manipulation and configures an OpenTracing-compatible tracer to capture
+and export trace spans. The agent also registers an OpenTracing `GlobalTracer`
+so you can add custom instrumentation to your application later.
```

Comment:
An aim of this is to support existing OT custom instrumentation as well as add on if none exists.